src/components/accordion/index.md.njk

```diff
@@ -98,11 +98,13 @@ If you’ve decided that you need the summary line, you must make it as short as
 
 The accordion component shows section headings as `<h2>` headings. If needed, change the heading level of the section headings to make them fit within the other headings on the page.
 
-### Change how an accordion starts
+### Configure how an accordion looks when the user first sees it
```

Comment:
To me, 'how an according looks' is more about theming or styling than which sections are open, and not sure I'd look under this heading to find out how to do it. What about something like…

```suggestion
### Starting with sections open
```

Reply:
Looks good to me!